src/components/Schema/Schema.tsx

```diff
@@ -66,10 +69,10 @@ export class Schema extends React.Component<Partial<SchemaProps>> {
     const types = Array.isArray(type) ? type : [type];
     if (types.includes('object')) {
       if (schema.fields?.length) {
-        return <ObjectSchema {...(this.props as any)} />;
+        return <ObjectSchema {...{ ...(this.props as any), level }} />;
       }
     } else if (types.includes('array')) {
-      return <ArraySchema {...(this.props as any)} />;
+      return <ArraySchema {...{ ...(this.props as any), level }} />;
```

Comment:
can we just pass `level` property directly in order to avoid double object destruction?

Reply:
`<ArraySchema { ...(this.props as any)} level={level} />`

Reply:
same for above